.github/workflows/integration.yaml

```diff
@@ -1,50 +0,0 @@
-name: Integration
-
-on:
-  push:
-    branches:
-      - main
-  pull_request:
-  workflow_dispatch:
-  schedule:
-    #        ┌───────────── minute (0 - 59)
-    #        │  ┌───────────── hour (0 - 23)
-    #        │  │ ┌───────────── day of the month (1 - 31)
-    #        │  │ │ ┌───────────── month (1 - 12 or JAN-DEC)
-    #        │  │ │ │ ┌───────────── day of the week (0 - 6 or SUN-SAT)
-    #        │  │ │ │ │
-    #        │  │ │ │ │
-    #        │  │ │ │ │
-    #        *  * * * *
-    - cron: '30 1 * * *'
-
-jobs:
-  integration-test:
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        ruby-version: [2.7.x, 2.6.x, 2.5.x]
-    steps:
-      - uses: actions/checkout@v2
-      - name: Set up Ruby ${{ matrix.ruby-version }}
-        uses: actions/setup-ruby@v1
-        with:
-          ruby-version: ${{ matrix.ruby-version }}
-      - run: |
-          gem install bundler
-          bundle install
-      - run: rake test:integration:fluent
-
-  notify:
```

Comment:
Did you remove this intentionally?

Reply:
It is added to `checks.yaml`

Reply:
It doesn't run daily anymore because we have better tests in bb anyway for that